src/main/java/org/spectra/cluster/util/DefaultParameters.java

```diff
@@ -0,0 +1,95 @@
+package org.spectra.cluster.util;
+
+import lombok.Data;
+
+import java.io.*;
+import java.net.URISyntaxException;
+import java.util.Properties;
+
+/**
+ * This code is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * ==Overview==
+ *
+ * @author ypriverol on 18/10/2018.
+ */
+@Data
+public class DefaultParameters {
+
+    private String binaryDirectory;
+    private boolean reuseBinary;
+    private boolean fastMode;
+    private Integer clusterRounds;
+    private boolean filterReportPeaks;
+    private Integer numberHigherPeaks;
+    private Double precursorIonTolerance;
+    private Double fragmentIonTolerance;
+
+    private Float thresholdStart;
+    private Float thresholdEnd;
+    private int nInitiallySharedPeaks;
+
+
+    public DefaultParameters(){
+
+        try {
+            Properties properties = readProperties();
+            setProperties(properties);
+
+        } catch (URISyntaxException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    private void setProperties(Properties properties) {
+        if(properties.containsKey("precursor.tolerance"))
```

Comment:
Wouldn't it be better to directly use the CliOptions.Options values here instead of the actual string?